src/cpu/aarch64/jit_uni_batch_normalization.cpp

```diff
@@ -2001,8 +2001,10 @@ struct jit_bnorm_t : public jit_generator {
         }
         if (jbp_->is_nspc_) {
             // comeback
-            if (!pd_->use_global_stats())
+            if (!pd_->use_global_stats()) {
                 LDR_ASSERT(reg_src, sp, (int)stack_off_src);
```

Comment:
I am a fan of braced-ifs anyway but it would be nice if the macro's themselves used the `do { ... } while(0)` idiom to guard against this happening in the future.